Assets/Scripts/Game/WeatherManager.cs

```diff
@@ -283,22 +296,45 @@ void SetSunlightScale()
         {
             // Start with default scale
             float scale = SunlightManager.defaultScaleFactor;
+            float shadowStrength = SunlightManager.defaultShadowStrength;
 
             // Apply winter
             if (_dfUnity.WorldTime.Now.SeasonValue == DaggerfallDateTime.Seasons.Winter)
+            {
                 scale = WinterSunlightScale;
+                shadowStrength = WinterShadowStrength;
+            }
 
             // Apply rain, storm, snow light scale
             if (IsRaining && !IsStorming)
+            {
                 scale = RainSunlightScale;
+                shadowStrength = RainShadowStrength;
+            }
             else if (IsRaining && IsStorming)
+            {
                 scale = StormSunlightScale;
+                shadowStrength = StormShadowStrength;
+            }
             else if (IsSnowing)
+            {
                 scale = SnowSunlightScale;
+                shadowStrength = SnowShadowStrength;
+            }
+            else if (IsOvercast)
+            {
+                scale = OvercastSunlightScale;
+                shadowStrength = OvercastShadowStrength;
+            }
+
+            shadowStrength *= Mathf.Exp(-50f * currentOutdoorFogSettings.density);
```

Comment:
I could have added shadowStrength to FogSettings, but that would have broken the compatibility with Enhanced Sky...
So I used a formula instead